.github/workflows/botonic-cli-tests.yml

```diff
@@ -30,3 +30,72 @@ jobs:
       run: (cd ./packages/$PACKAGE && npm run build)
     - name: Verify lint
       run: (cd ./packages/$PACKAGE && npm run lint_ci)
+  botonic-cli-templates-tutorial-tests:
+    name: Botonic cli template tests
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE: botonic-cli
+    steps:
+      - name: Checking out to current branch
+        uses: actions/checkout@v1
+      - name: Setting up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```

Comment:
should we setup it with v10?

Reply:
why? 12 is the lts and our target version